core/client/fs/src/main/java/alluxio/client/file/cache/CacheEvictor.java

```diff
@@ -24,10 +27,34 @@ static CacheEvictor create() {
     return null;
   }
 
-  void updateOnGet(long pageId);
+  /**
+   * Updates evictor after a get operation.
+   *
+   * @param fileId ID of the file
+   * @param pageIndex index of the page within the file
+   */
+  void updateOnGet(long fileId, long pageIndex);
 
-  void updateOnPut(long pageId);
+  /**
+   * Updates evictor after a put operation.
+   *
+   * @param fileId ID of the file
+   * @param pageIndex index of the page within the file
+   */
+  void updateOnPut(long fileId, long pageIndex);
 
-  List<Long> getPagesToEvict(int pages);
+  /**
+   * Updates evictor after a delete operation.
+   *
+   * @param fileId ID of the file
+   * @param pageIndex index of the page within the file
+   */
+  void updateOnDelete(long fileId, long pageIndex);
 
+  /**
+   * Find a page to evict.
+   *
+   * @return a pair of long values representing (fileId, pageIndex)
+   */
+  Pair<Long, Long> evict();
```

Comment:
Given how often we use the `fileId, pageIndex` pair, it might be worth creating an explicit object.

Reply:
Refactored.